src/pages/DispensingPage.js

```diff
@@ -109,7 +118,10 @@ const Dispensing = ({
             value={searchTerm}
             viewStyle={localStyles.searchBar}
           />
-          <PageButton text="New Patient" onPress={createPatient} />
+          <PageButton
+            text={dataSet === DATA_SET.PATIENTS ? 'New Patient' : 'New Prescriber'}
```

Comment:
localisation again. is the plan to add to the #1693 issue still?

Reply:
yeah, sorry, added - will do a localization round after these latest PR's are in and I update from develop (which had some changes to how localization works)